plugins/references.py

```diff
@@ -69,7 +71,7 @@ def getMarkers(view, name=""):
             markers.extend(view.find_all(r"(?<=\[)(%s)(?=\])(?!\s*\]:)" % name, 0))
     regions = []
 
-    selector = marker_text_scope_name + ", " + marker_text_scope_name
+    selector = marker_text_scope_name + ", " + marker_link_reference_scope_name
```

Comment:
The selector `selector = marker_text_scope_name + ", " + marker_text_scope_name` contains duplicate scope names by accident. It should rather look like `selector = marker_ref_scope_name + ", " + marker_text_scope_name` to match the original behavior of the current and recent releases.

The `marker_ref_scope_name` would already match the name within brackets, only. I guess we therefore don't need to introduce another marker scope name?
    


My selector was wrong. It needs to look like